virttest/utils_qemu.py

```diff
@@ -160,3 +160,19 @@ def get_maxcpus_hard_limit(bin_path, machine_type):
     except KeyError:
         raise ValueError("Could not get the maximum limit CPUs supported by "
                          "this machine '%s'" % machine_type)
+
+
+def get_dev_attrs(bin_path, opt_name, dev_name):
+    """
+    Get the device attributes from qemu help doc
+
+    :param bin_path: Path to qemu binary
+    :param opt_name: qemu option name
+    :param dev_name: qemu device name
+    :return: attributes list
+    """
+    help_doc = process.run("%s -%s %s,?" % (bin_path, opt_name, dev_name),
```

Comment:
A bit risk that machine type is must in some situation.    `execute_qemu` fix it while it is a to do in `utils_qemu`
refer:
https://github.com/avocado-framework/avocado-vt/blob/772e0d2eaff1c745d5603bd8cae97ce64a93a4ff/virttest/qemu_devices/qcontainer.py#L628
https://github.com/avocado-framework/avocado-vt/blob/772e0d2eaff1c745d5603bd8cae97ce64a93a4ff/virttest/qemu_devices/qcontainer.py#L148

https://github.com/avocado-framework/avocado-vt/blob/772e0d2eaff1c745d5603bd8cae97ce64a93a4ff/virttest/utils_qemu.py#L81

Reply:
I doubt if machine type is a must.  According to @zhenyzha , the test passed on arm. @zhenyzha  Would you pls help check if machine type is must on arm? Thanks.

Reply:
Yes, i saw the arrch64 test result already,  the latest status may have changed, can work without it.  However, not confident about other usage, so i think it is better to add a TBD too if no plan to fix it with this patch.

Reply:
How about add '-machine none' to the cmd? Device attrs should have nothing to do machine types.

Reply:
Yes,  i doesn't see the risk to add '-machine none' for all situation to check dev attribute.  so, ok for me.
@YongxueHong  Any comments with this ? Thanks!

Reply:
> Yes, i doesn't see the risk to add '-machine none' for all situation to check dev attribute. so, ok for me. @YongxueHong Any comments with this ? Thanks!

Nope, so let's merge it.
Thanks all.